web/src/views/LoginPortal/FirstFactor/FirstFactorForm.tsx

```diff
@@ -28,6 +28,12 @@ export default function (props: Props) {
     const [password, setPassword] = useState("");
     const [passwordError, setPasswordError] = useState(false);
     const { createErrorNotification } = useNotifications();
+    const usernameRef = useRef() as MutableRefObject<HTMLInputElement>;
+    const passwordRef = useRef() as MutableRefObject<HTMLInputElement>;
+    useEffect(() => {
+        const timeout = setTimeout(() => usernameRef.current.focus(), 10);
```

Comment:
As discussed I will have a look at how we can better fix it. In the meantime we use that PR to merge in the change of error message and the auto focus of the OTP field. We'll do the username field in another PR.